frontend/src/app/main/ui/workspace/libraries.cljs

```diff
@@ -400,51 +464,205 @@
                           (let [library-id (some-> (dom/get-target event)
                                                    (dom/get-data "library-id")
                                                    (parse-uuid))]
+                            (reset! updating?* true)
                             (st/emit! (dwl/sync-file file-id library-id)))))]
+
     (if new-css-system
       [:div {:class (css :section)}
-       (if (empty? libraries)
+       (if (empty? libs-assets)
          [:div {:class (css :section-list-empty)}
           (tr "workspace.libraries.no-libraries-need-sync")]
          [:*
-          [:div {:class (css :section-title)} (tr "workspace.libraries.library")]
+          [:div {:class (css :section-title)} (tr "workspace.libraries.library-updates")]
 
           [:div {:class (css :section-list)}
-           (for [{:keys [id name] :as library} libraries]
+           (for [[{:keys [id name] :as library}
+                  exceeded
+                  {:keys [components colors typographies]}] libs-assets]
              [:div {:class (css :section-list-item)
                     :key (dm/str id)}
               [:div
                [:div {:class (css :item-name)} name]
-               [:div {:class (css :item-contents)} (describe-external-library library)]]
-              [:input {:class (css :item-update)
-                       :type "button"
+               [:div {:class (css :item-contents)} (describe-library
+                                                    (count components)
+                                                    0
+                                                    (count colors)
+                                                    (count typographies))]]
+              [:input {:type "button"
+                       :class (dom/classnames (css :item-update) true
```

Comment:
a new css, high performance macros should be used for new code, instead of `dom/classnames`

Reply:
done